mollie/api/resources/customer_subscriptions.py

```diff
@@ -8,15 +8,28 @@ class CustomerSubscriptions(Base):
     customer_id = None
 
     def get_resource_object(self, result):
-        return Subscription(result)
+        subscription = Subscription(result)
+        subscription._resource = self
+        return subscription
 
     def get(self, subscription_id, **params):
         if not subscription_id or not subscription_id.startswith(self.RESOURCE_ID_PREFIX):
             raise Error(
-                'Invalid subscription ID: "%s". A subscription ID should start with "%s".' % (subscription_id, self.RESOURCE_ID_PREFIX)
+                'Invalid subscription ID: "%s". A subscription ID should start with "%s".' % (
+                subscription_id, self.RESOURCE_ID_PREFIX)
             )
         return super(CustomerSubscriptions, self).get(subscription_id)
 
+    def delete(self, subscription_id):
+        """Override the delete function of base.py so it returns a response"""
```

Comment:
Voorbeeld: "Deleting a subscription causes the subscription status to changed to 'canceled'. The updated subscription object is returned."